index.js

```diff
@@ -98,15 +93,27 @@ var Frame = React.createClass({
       setTimeout(this.renderFrameContents, 0);
     }
   },
+  getMountTarget: function() {
+    var doc = this.getDoc();
+
+    if(this.props.mountTarget) {
+      return doc.querySelector(this.props.mountTarget);
+    }
+
+    return doc.body.children[0];
+  },
+  getDoc: function() {
+    return ReactDOM.findDOMNode(this).contentDocument;
+  },
   componentDidUpdate: function() {
     this.renderFrameContents();
   },
   componentWillUnmount: function() {
     this._isMounted = false;
 
-    var doc = ReactDOM.findDOMNode(this).contentDocument;
+    var doc = this.getDoc();
     if (doc) {
-      ReactDOM.unmountComponentAtNode(doc.body);
+      ReactDOM.unmountComponentAtNode(this.getMountTarget());
```

Comment:
Can you add a test for this path to make sure `this.getMountTarget` returns what you expect when `mountTarget` prop is set.